### ISSUE TITLE
Stack Report fails for VSCode v1.25.1 on Linux

### ISSUE THREAD

**Issue by sara-02** [fabric8-analytics/fabric8-analytics-vscode-extension#138]
On VScode 1.25 the stack report generation is failing with the error
```js
 Failed to trigger stack analysis: 
/usr/share/code/resources/app/out/vs/workbench/workbench.main.js:2364 
```
VSCode Details
```
Version: 1.25.1
Commit: 1dfc5e557209371715f655691b1235b6b26a06be
Date: 2018-07-11T15:40:20.190Z
Electron: 1.7.12
Chrome: 58.0.3029.110
Node.js: 7.9.0
V8: 5.8.283.38
Architecture: x64
```
The component analysis is working as expected for the same.
![screenshot from 2018-07-24 17-57-15](https://user-images.githubusercontent.com/7047079/43137955-0381add4-8f6b-11e8-8455-4fefff569b24.png)

**Comment by sara-02** [comment]
Update: Tried the same set of extensions with VSCode 1.19.3 and it worked
![screenshot from 2018-07-24 18-19-59](https://user-images.githubusercontent.com/7047079/43139005-448c766c-8f6e-11e8-9196-6d502d2867f2.png)

**Comment by sara-02** [comment]
But it looks like component analysis did not work because the stack report shows a CVE but in the console it says `No problems have been detected in the workspace so far.`

**Comment by invincibleJai** [comment]
thanks @sara-02 for reporting it :)

**Comment by sara-02** [comment]
Closing this as it is fixed by #139
